letsdrill.rb

```diff
@@ -1,24 +1,33 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
 
+  if integer >= 90
+    return "A"
+  elsif integer >= 80 and integer < 90
+    return "B"
+  elsif integer >= 70 and integer < 80
+    return "C"
+  elsif integer >= 60 and integer < 70
+    return "D"
+  else
+    return "F"
```

Comment:
Rubyists tend to prefer implicitly returning from methods rather than calling `return`. https://github.com/bbatsov/ruby-style-guide#no-explicit-return

This method should work the same after deleting all the "return"s.